src/external.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2018-2020, Michael P. Howard
+# Copyright (c) 2021-2022, Auburn University
+# This file is part of the azplugins project, released under the Modified BSD License.
+
+
+import hoomd
+
+from hoomd.azplugins import _azplugins
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.data.typeconverter import OnlyTypes, variant_preprocessing
+from hoomd.md import force
+
+class HarmonicBarrier(force.Force):
+
+    _ext_module = _azplugins
+#    _cpp_class_name = "PlanarHarmonicBarrier"
```

Comment:
These are not needed in our own plugin. Specify the module explicitly when you return the class name
```suggestion
```